go/tools/builders/cc.go

```diff
@@ -13,12 +13,13 @@ import (
 func cc(args []string) error {
 	cc := os.Getenv("GO_CC")
 	if cc == "" {
-		errors.New("GO_CC environment variable not set")
```

Comment:
this looks like just an error in the original code, `errors.New`here did nothing and was effectively just ignoring the error.

Reply:
Nice catch. I think that you can just return the error here instead. I wonder why nogo didn't catch this.

Reply:
fixed